### ISSUE TITLE
Implement USB pass-through in KVM

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#334]
Originally reported of Google Code with ID 278.

```
Experimental patch:
https://groups.google.com/d/msg/ganeti/uON4JFAZEP4/5PwOmeJ6MUQJ
```
Originally added on 2012-10-16 10:09:31 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in:

156681c kvm: support usb devices

```
Originally added on 2013-02-26 15:56:50 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
This has been fixed in:

156681c kvm: support usb devices

```
Originally added on 2013-02-26 15:56:51 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
